scripts/setup.sh

```
@@ -0,0 +1,503 @@
+    else
+        pwd
+    fi
+}
+
+SCRIPT_DIR="$(get_script_dir)"
+
+# Try to source common helpers
+# shellcheck source=/dev/null
+COMMON_HELPERS="$SCRIPT_DIR/common-codex.sh"
+
+if [ -f "$COMMON_HELPERS" ]; then
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)